node-syscall/syscall_test.go

```diff
@@ -0,0 +1,36 @@
+// +build js
+
+package syscall_test
```

Comment:
The node-syscall package is for the node-syscall C code. It's not meant to be a Go package. As a result, this Go test will be not be run by anything.

We have the `tests` directory for standalone tests of things that aren't in the Go standard library. Please move this test there. It can be a new directory or a new file there.